resources/assets/js/components/MyGlobleSetting.js

```diff
@@ -0,0 +1,6 @@
+class MyGlobleSetting {
```

Comment:
Maybe you mean `MyGlobalSettings`?

Reply:
It's wrong spelling, so can you fix ? Global, not Globale

Reply:
i have fix it